### PR TITLE
Add new data from manga

### PR DESCRIPTION
**Pull request by correac**
Data published in Yates et al. (2021) (https://arxiv.org/pdf/2011.04670.pdf). I rebased the recent changes from Evgenii's.

### CODE REVIEW

File: data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+comment = (
+    "Data obtained from Yates et al. (2021). No need for "
+    "h-correction. The metallicity is expressed in units of solar metallicity, using Z=0.0142. "
+    f"This has been corrected to use Z_solar={solar_metallicity}. "
+)
+citation = "Yates et al. (2021, MaNGA)"
+bibcode = "2021MNRAS...503...4474Y"
+name = "Stellar mass - Stellar metallicity relation"
+plot_as = "points"
+redshift = 0.1
```
Comment (EvgeniiChaikin) [inline data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py:43]: Shouldn't it be `z=0` (according to the paper)?

File: data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+Z_median = 10 ** raw[:, 1] * unyt.dimensionless * Z_solar_obs / solar_metallicity
+Z_std_lo = 10 ** raw[:, 2] * unyt.dimensionless * Z_solar_obs / solar_metallicity
+Z_std_hi = 10 ** raw[:, 3] * unyt.dimensionless * Z_solar_obs / solar_metallicity
+
+y_scatter = unyt.unyt_array((Z_median - Z_std_lo, Z_std_hi - Z_median))
+
+
+# Meta-data
+comment = (
+    "Data obtained from Yates et al. (2021). No need for "
```
Comment (EvgeniiChaikin) [inline data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py:35]: The paper says that these data are for star-forming, disc-dominant galaxies. Let's include this information in the comment.

**Review by EvgeniiChaikin — APPROVED** [review]
Thank you @correac!

All looks very good! 

I added two small comments to clarify a couple of things. Otherwise, the PR is ready to be merged.

File: data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py (hunk 1 above)
Comment (correac) [reply data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py]: Yes, added.

File: data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py (hunk 2 above)
Comment (correac) [reply data/GalaxyStellarMassStellarMetallicity/conversion/convertYates2021.py:35]: I included it in the comment. That info is also included in the caption (see pipeline config pull request).

**Pull request merged**